README.md

```diff
@@ -2,15 +2,15 @@
 
 logr offers an(other) opinion on how Go programs and libraries can do logging
 without becoming coupled to a particular logging implementation,  This is not
```

Comment:
Should this comma be a period? Two spaces and the upper-case T seem to indicate that it should.

Reply:
Yep.  That's a good catch.  Fixed.

I also went through the rest of the file and found a few other punctuation and gramer issues.

Thanks for looking.